starter-code/client/src/pages/ride-files/ListObject.js

```diff
@@ -0,0 +1,45 @@
+import React, {Component} from 'react'
+
+class ListObject extends Component {
```

Comment:
Can we be consistent on class components and functional components? I think all our other components are functional components.